airbyte-cdk/bulk/core/load/src/main/kotlin/io/airbyte/cdk/load/task/DestinationTaskLauncher.kt

```diff
@@ -86,13 +80,16 @@ class DestinationTaskLauncher(
     private val syncManager: SyncManager,
 
     // Internal Tasks
-    private val inputConsumerTaskFactory: InputConsumerTaskFactory,
+    private val inputConsumerTask: InputConsumerTask,
```

Comment:
A task feels like an ephemeral concept, why is this part of the `DestinationTaskLauncher` state?

Reply:
maybe a naming thing - iirc Task is basically a Runnable; this class just orchestrates which things run and at what time